src/lib/query.ts

```diff
@@ -1949,16 +1995,32 @@ public static async queryPriceRuleActions (conn: Connection): Promise<String[]>
 public static async queryChargeElementStdPricebookEntries (conn: Connection, productList: Set<String>): Promise<String[]> {
         Util.log('--- exporting Charge Elements standard PricebookEntry ');
         if(productList.size >90){
-            let paramsObject: Query={     
-                "queryBegining": "SELECT Pricebook2.enxCPQ__TECH_External_Id__c, Product2.enxCPQ__TECH_External_Id__c, CurrencyIsoCode, Pricebook2Id, Product2Id," + this.pbeQuery +" FROM PricebookEntry WHERE Product2.enxCPQ__Root_Product__r.Name IN (",
-                "queryConditions": ") AND Pricebook2.IsStandard = true AND Product2.RecordType.Name = 'Charge Element'",
-                "objectsList": productList,
-                "sobjectName": "Charge Elements standard PricebookEntry"
+            let paramsObject: Query;
+            if(!this.currenciesList){
+                paramsObject={     
+                    "queryBegining": "SELECT Pricebook2.enxCPQ__TECH_External_Id__c, Product2.enxCPQ__TECH_External_Id__c, CurrencyIsoCode, Pricebook2Id, Product2Id," + this.pbeQuery +" FROM PricebookEntry WHERE Product2.enxCPQ__Root_Product__r.Name IN (",
+                    "queryConditions": ") AND Pricebook2.IsStandard = true AND Product2.RecordType.Name = 'Charge Element'",
+                    "objectsList": productList,
+                    "sobjectName": "Charge Elements standard PricebookEntry"
+                }
+                return await Util.createQueryPromiseArray(paramsObject, conn);
+            }else{
+               paramsObject={     
+                    "queryBegining": "SELECT Pricebook2.enxCPQ__TECH_External_Id__c, Product2.enxCPQ__TECH_External_Id__c, CurrencyIsoCode, Pricebook2Id, Product2Id," + this.pbeQuery +" FROM PricebookEntry WHERE Product2.enxCPQ__Root_Product__r.Name IN (",
+                    "queryConditions": ") AND Pricebook2.IsStandard = true AND Product2.RecordType.Name = 'Charge Element' AND CurrencyIsoCode IN (" + Util.setToIdString(this.currenciesList) + ") ",
+                    "objectsList": productList,
+                    "sobjectName": "Charge Elements standard PricebookEntry"
+                }
+                return await Util.createQueryPromiseArray(paramsObject, conn);
```

Comment:
Hide it.
Also I didn't notice it before, but isn't the difference in additional "AND" in queryConditions? If so maybe we could simplify that a little bit more?